app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +14,11 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def role_allows_feature?
+    return true if self[:child_id].nil?
```

Comment:
but the problem remains, right? if `:child_id` is nil (not a button under a drop-down) then the rbac check is skipped and true is returned